target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,10 @@ board_config_update
 board="$(cat /tmp/sysinfo/board_name)" 2>/dev/null
 
 case "$board" in
+pc-engines-apu2)
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
+	ucidef_set_interface_raw "opt" "eth2"
```

Comment:
@obsy This is done mainly to match other firmware's that run on the device, ex PfSense. If there is a good argument to merge them instead then that path could be taken as well.